src/stash.c

```diff
@@ -476,6 +497,20 @@ void stash_group_add_boolean(StashGroup *group, gboolean *setting,
 }
 
 
+/** Adds double setting.
+ * @param group .
+ * @param setting Address of setting variable.
+ * @param key_name Name for key in a @c GKeyFile.
+ * @param default_value Value to use if the key doesn't exist when loading. */
+GEANY_API_SYMBOL
+void stash_group_add_double(StashGroup *group, gdouble *setting,
+		const gchar *key_name, gdouble default_value)
+{
+	gulong *default_long = (gulong*) &default_value;
+	add_pref(group, G_TYPE_DOUBLE, setting, key_name, (gpointer) *default_long);
```

Comment:
add_pref should use a union. I'll prepare a change, then you can rebase this.

Reply:
Will rebase when it's available.  Thank you.